cfme/tests/containers/test_containers_data_integrity_properties.py

```diff
@@ -0,0 +1,107 @@
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from cfme.web_ui import CheckboxTable
+from cfme.containers.image import Image
+from cfme.containers.image_registry import ImageRegistry
+from cfme.containers.pod import Pod
+from cfme.containers.provider import Provider
+from cfme.containers.node import Node
+from cfme.containers.container import Container
+from utils import testgen
+from utils.version import current_version
+from cfme.web_ui import history
+
+list_tbl = CheckboxTable(table_locator="//div[@id='list_grid']//table")
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope='function')
+
+container_image_fields = ['Name', 'Image Id', 'Full Name']
+container_nodes_fields = ['Name', 'Creation timestamp', 'Resource version', 'Number of CPU Cores',
+                          'Memory', 'Max Pods Capacity', 'System BIOS UUID', 'Machine ID',
+                          'Infrastructure Machine ID', 'Runtime version', 'Kubelet version',
+                          'Proxy version', 'Operating System Distribution', 'Kernel version']
+container_fields = ['Name', 'State', 'Restart count', 'Backing Ref (Container ID)', 'Privileged']
+image_registry_field = ['Host']
+
+
+# CMP-9945
+def test_containers_integrity_properties():
+    """ Properties fields test in Containers
+        This test checks correct population of the Properties Fields in Containers' details menu
+        Steps:
+            * Goes to Containers -- > Containers menu
+            * Go through each Container in the menu and check validity of Properties fields
+        """
+    sel.force_navigate('containers_containers')
+    container_name = [r.name.text for r in list_tbl.rows()]
+    container_pod_name = [r.pod_name.text for r in list_tbl.rows()]
+    dictionary = dict(zip(container_name, container_pod_name))
```

Comment:
Yeah, I have a strong illusion that all the Container PRs add the same code when I am looking at them so I am lost in that. Pete made a good point. What repeats, needs DRY :)